templates/base.html

```diff
@@ -48,8 +50,8 @@
     </ul>
 
     <ul class="nav navbar-nav navbar-right">
-       </li>
-      <li class="dropdown" ><a href="{{base_url}}/feedback">Feedback</a>
+       
+      <li class="dropdown"><a href="http://127.0.0.1:8097//feedback" style="padding-left: 40px;">Feedback</a>
```

Comment:
the href value is hardcoded. Revert code here.

Reply:
Changed it back and successfully tested it, so it should be working.